astro/src/styles/bootstrap.scss

```diff
@@ -165,37 +182,77 @@ $utilities: map-merge(
 }
 
 .hero-link:focus-visible, .hero-button:focus-visible {
-  outline-color: white;
+  outline-color: #FFFFFF;
 }
 
 .form-field:focus {
   border-style: solid;
 }
 
+.breadcrumb-item, .breadcrumb-item.active, .nav-link, .dropdown-item, .dropdown-item:hover {
+  color: #000000;
+}
+
+.dropdown-item:hover {
+  background-color: $body-tertiary-bg;
+}
+
 .navbar-brand, .text-brand {
   font-family: $headings-font-family;
 }
 
+.card, .card-header {
+  background-color: $body-secondary-bg;
+}
+
+textarea, input {
+  border-width: 1.5px 1.5px;
+  border-color: #000000;
+}
+
+input[type=checkbox], input[type=radio] {
+  accent-color: #041058;
+}
+
 img.profile-pic {
   display: inline-block;
   max-width: 3em;
 }
 
 @include color-mode(dark) {
-  *:focus-visible, .form-field:focus, .toggle-menu:focus, .navbar-toggler:focus, .btn:focus-visible {
-    outline-color: white;
+  *:focus-visible, .form-field:focus, .toggle-menu:focus, .navbar-toggler:focus, .btn:focus-visible, .nav-link:focus-visible {
+    outline-color: #FFFFFF;
   }
 
   .text-ac {
     color: var(--bs-light) !important;
   }
+
+  .border-ac {
+    border-color: var(--bs-light) !important;
+  }
+
   .link-ac {
     color: var(--bs-light) !important;
     text-decoration-color: RGBA(var(--bs-light-rgb), var(--bs-link-underline-opacity, 1)) !important;    
   }
+
+  .shadow, .shadow-lg {
+    box-shadow: none !important;
+  }
+
+  .breadcrumb-item, .breadcrumb-item.active, .nav-link, .dropdown-item, .dropdown-item:hover {
+    color: #FFFFFF;
+  }
+
+  textarea, input {
+    border-color: #FFFFFF;
+  }
+
   h1, h2, h3, h4, h5, h6 {
     color: var(--bs-primary-text-emphasis);
   }
+
   .bi {
```

Comment:
Rachael noted that the blue color generated here doesn't quite match the blue color on the headings in dark mode. Let's try to make these the same (or at least very similar / on the same color palette).